src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -38,54 +44,67 @@ const styles = (theme: Theme) =>
     noAlert: {
       opacity: '50%',
     },
-    sublabel: {
-      color: '#BDBDBD',
-    },
-    icon: {
-      width: '20px',
-      height: '20px',
+    badgeStyles: {
+      height: 24,
+      width: 24,
       color: theme.palette.common.white,
       display: 'flex',
       alignItems: 'center',
       justifyContent: 'center',
       borderRadius: '50%',
-      backgroundColor: '#FF6142',
+      backgroundColor: theme.palette.error.main,
+    },
+    checked: {
+      backgroundColor: theme.palette.primary.main,
     },
-    grey: {
-      backgroundColor: '#BDBDBD',
+    innerContent: {
+      display: 'flex',
+      justifyContent: 'flex-start',
+      alignItems: 'center',
     },
   });
 
 function HomeMenuItem(props: HomeMenuItemProps) {
-  const { label, sublabels, noBadge, classes } = props;
-  const amount = props.amount
-    ? sublabels
-      ? sublabels.reduce(function (amt: number, curr: sublabel) {
-          return amt + curr.amount;
-        }, 0)
-      : props.amount
-    : null;
+  const { label, noBadge, classes, iconType, amount } = props;
+  const renderIcons = () => {
+    if (iconType === 'meter' || iconType === 'collect') {
+      return (
+        <AccountBoxIcon fontSize='large' color="primary" />
+      );
+    }
+    if (iconType === 'maintenance') {
+      return (
+        <BuildIcon fontSize='large' color="primary" />
+      );
+    }
+    if (iconType === 'incident') {
+      return (
+        <WarningIcon fontSize='large' color="primary" />
+      );
+    }
+  }
+
+  const getRightBadge = () => {
+    if (amount === 0) {
+      return <CheckCircleIcon color="primary" />
+    }
+
+    // We cap at 99 so the circle spacing doesn't break
+    return <div className={classes.badgeStyles}>
+      <Typography variant="body2">{amount > 99 ? '99+' : amount}</Typography>
+    </div>
+  }
```

Comment:
This seems to be working fine so feel free to ignore this, but you could switch to the MUI badge component https://material-ui.com/components/badges/#badge which takes care of this kind of behavior of '99+', as well as resizing to accommodate length etc.

Reply:
I tried it briefly with badge before switching to this but couldn't get it to look as nice. I'll put a TODO in case they want to fix the 99+ behavior / resizing